proto/cosmos/gov/v1/gov.proto

```diff
@@ -12,18 +12,45 @@ import "amino/amino.proto";
 
 option go_package = "cosmossdk.io/x/gov/types/v1";
 
+// ProposalType enumerates the valid proposal types.
+// All proposal types are v1.Proposal which have different voting periods or tallying logic.
+enum ProposalType {
+  // PROPOSAL_TYPE_UNSPECIFIED defines no proposal type, which fallback to PROPOSAL_TYPE_STANDARD.
+  PROPOSAL_TYPE_UNSPECIFIED = 0;
+  // PROPOSAL_TYPE_STANDARD defines the type for a standard proposal.
+  PROPOSAL_TYPE_STANDARD = 1;
+  // PROPOSAL_TYPE_MULTIPLE_CHOICE defines the type for a multiple choice proposal.
+  PROPOSAL_TYPE_MULTIPLE_CHOICE = 2;
+  // PROPOSAL_TYPE_OPTIMISTIC defines the type for an optimistic proposal.
+  PROPOSAL_TYPE_OPTIMISTIC = 3;
+  // PROPOSAL_TYPE_EXPEDITED defines the type for an expedited proposal.
+  PROPOSAL_TYPE_EXPEDITED = 4;
+}
+
 // VoteOption enumerates the valid vote options for a given governance proposal.
 enum VoteOption {
```

Comment:
this seems like it could cause issues for clients, have you looked into how it will work for wallets that depend on this?

Reply:
which one is the alias here? im worried this is a proto feature that no one will understand unless they read the docs but we already assume that people wont do it. Can we only have the option votes?

Reply:
The option votes are the one used by default (https://github.com/cosmos/cosmos-sdk/blob/84a2e4bb03193fbf62259ad11f8bcfb98531c441/x/gov/types/v1/gov.pb.go#L102-L126), but clients can still submit tx using both for backward compatibility. It would have been breaking without this but now it isn't.